frontend/src/components/Collections/components/Dataset/components/DownloadDataset/index.tsx

```diff
@@ -17,7 +18,7 @@ const DownloadDataset: FC<Props> = ({
   isDisabled = false,
   name,
 }) => {
-  const [isOpen, setIsOpen] = React.useState(false);
+  const { onClose, onOpen, open } = useDialog();
```

Comment:
Collection view dialog components now use a basic `useDialog` hook that controls dialog state and functionality i.e. `open`, `onClose` (refer [shared hook](https://github.com/chanzuckerberg/single-cell-data-portal/pull/7234/files#diff-b25d215d4b7fd16cceae5090e40f467bb6b46e838e05cfa1d90c06bb2db15b67R13)).